package.json

```diff
@@ -105,9 +105,13 @@
     "gulp": "^4.0.2",
     "gulp-filter": "^5.1.0",
```

Comment:
The only break change on this dep was the drop support for the Node versions below 8. WDYT?

Reply:
Not sure what you're referring to. Final changelog has no changes in marked lines.

Reply:
No problem, it's your call.